### ISSUE TITLE
Vue Router documentation with Dark reader is not working

### ISSUE THREAD

**Issue by jamesli2021** [vuejs/router#1598]
### Version
4.1.6

### Reproduction link
[https://example.com/](https://example.com)

### Steps to reproduce
Open:
https://router.vuejs.org/

### What is expected?
Background should appear dark or black with Dark Reader for accessibility experience.

I have eye cataract and feel the white background is  huge pain my eye sight.

### What is actually happening?
The page is appear with white background.

**Comment by posva** [comment]
This will be added once we can upgrade vitepress
